### DISCUSSION TITLE
How to convert canvas to base64 use js I need Store DB

### DISCUSSION THREAD

**Discussion by Ali-Hassan-Ali** [pixijs/pixijs#7871] [Q&A] [answered]
How to convert your canvas into an image?

**Comment by Ali-Hassan-Ali** [comment]
Help Me

**Comment by domis86** [comment, marked as answer]
> Help Me

For such requests i would advice to:
* use "Discussion" ( https://github.com/pixijs/pixijs/discussions ) instead of Issue
* try to explain your problem with as much relevant details as possible in description:
    * what you want to achieve?
    * what have you tried so far?

**Reply by Ali-Hassan-Ali** [reply]
How to convert your canvas into an image?

**Comment by ivanpopelyshev** [comment]
`renderer.extract.base64()`

**Reply by Ali-Hassan-Ali** [reply]
how to use code please example | renderer.extract.base64()

**Reply by ivanpopelyshev** [reply]
Sorry, my beliefs prohibit me from answering on easy questions. You have to gather some faith and look in the documentation. https://pixijs.download/dev/docs/ . But basically, that's just it : `let myBase64 = app.renderer.extract.base64();`

**Reply by Ali-Hassan-Ali** [reply]
😉 : Thanks, I will do your advice | You are such a wonderful person
